structure/assign_gn_foldseek.py

```diff
@@ -0,0 +1,394 @@
+from django.conf import settings
```

Comment:
I would have preferred if you modified the assign_gn script to incorporate the foldseek gn assign. This is a duplicate file in a way. Can we integrate it into that script? What is unique about this script compared to the original? If nothing else, how about inheriting the class and only overwriting the functions that are different?

Reply:
Yes that shouldn't be there, I was just testing something, I will take it out.